sunpy/coordinates/frames.py

```diff
@@ -673,11 +673,11 @@ def is_visible(self, *, tolerance: u.m = 1*u.m):
     @classmethod
     @sunpycontextmanager
     @deprecated('6.0', alternative='sunpy.coordinates.screens.SphericalScreen')
-    def assume_spherical_screen(cls, center, only_off_disk=False):
+    def assume_spherical_screen(cls, center, radius=None, only_off_disk=False):
```

Comment:
```suggestion
    def assume_spherical_screen(cls, center, *, radius=None, only_off_disk=False):
```

Reply:
To avoid a disruptive API change:
```python
    def assume_spherical_screen(cls, center, only_off_disk=False, *, radius=None):
```

Reply:
Good point, I forgot about the other keyword.